content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -223,13 +232,24 @@ Use the `equals`, `isEmpty`, `every`, `some` and `none` operations to filter for
 - `none`: None of the items in the list can match the condition
 - `isSet` : Filter optional fields to include only results that have been set (either set to a value, or explicitly set to `null`). Setting this filter to `true` will exclude `undefined` results that are not set at all.
 
-For example, you can use `equals` to find products with photos that match specific URL paths:
+For example, you can use `equals` to find products with a specific list of photos (all `url`, `height` and `width` fields must match):
```

Comment:
I used your example - there doesn't seem to be a good way right now to do what the example was previously trying to do, see conversation here https://prisma-company.slack.com/archives/C02FZENHD4N/p1658831817291979